src/context.lisp

```diff
@@ -87,15 +87,17 @@
     ok))
 
 (defun make-context (&key (method nil method-supplied-p)
-                          (disabled-protocols)
-                          (options (list +SSL-OP-ALL+))
-                          (session-cache-mode +ssl-sess-cache-server+)
-                          (verify-location :default)
-                          (verify-depth 100)
-                          (verify-mode +ssl-verify-peer+)
-                          (verify-callback nil verify-callback-supplied-p)
-                          (cipher-list +default-cipher-list+)
-                          (pem-password-callback 'pem-password-callback))
+                       (disabled-protocols)
+                       (options (list +SSL-OP-ALL+))
+                       (min-ssl-version +ssl3-version+)
+                       (max-ssl-version 0)
+                       (session-cache-mode +ssl-sess-cache-server+)
+                       (verify-location :default)
+                       (verify-depth 100)
+                       (verify-mode +ssl-verify-peer+)
+                       (verify-callback nil verify-callback-supplied-p)
+                       (cipher-list +default-cipher-list+)
+                       (pem-password-callback 'pem-password-callback))
```

Comment:
Unneeded indentation change

Reply:
My apologies. Emacs did it. I'll undo it.